node/src/neighborhood/mod.rs

```diff
@@ -2061,7 +2071,14 @@ pub enum FallbackPreference {
     ExitCountryNoFallback,
 }
 
-//TODO create big comment about all members and its utilization
+// exit_countries contains all country_codes, that user selected for exit location and are present in DB,
+// these are used to stop recursion in routing_engine
+// fallback_preference is enum, that controls whether we want to strictly prohibit exit_location to nodes
+// with requested country_codes, or we accept other locations in case requested country is unavailable
+// locations_opt is Optional Vec of ExitLocation, it is set to Some(Vec<ExitLocation>) from users input,
```

Comment:
Tady se mozna zbytecne opakujes. Muzes nechat jen jednu cast, treba: "locations_opt is an optional vector of ExitLocation set from user's input".